openai.chat/types.bal

```diff
@@ -123,7 +122,7 @@ public type CreateChatCompletionRequest record {|
     # Modify the likelihood of specified tokens appearing in the completion.
     # 
     # Accepts a json object that maps tokens (specified by their token ID in the tokenizer) to an associated bias value from -100 to 100. Mathematically, the bias is added to the logits generated by the model prior to sampling. The exact effect will vary per model, but values between -1 and 1 should decrease or increase likelihood of selection; values like -100 or 100 should result in a ban or exclusive selection of the relevant token.
-    record {} logit_bias?;
+    record {}? logit_bias = ();
```

Comment:
Is this change correct

Reply:
Yeah this behaviour is the correct one, as the counterpart contract property is having `nullable: true`

Reply:
But this will cause an error when we send the request to the OpenAI. Shall we remove nullable: true from the contract

Reply:
yeah that seems to be an issue either in their contract or the service backend, as in the contract they expect null values even though they don't accept the same when it comes to the actual service

Reply:
+1 t remove the `nullable: true` from the contract file for now